server/protect.ts

```diff
@@ -11,3 +13,16 @@ export function protect(handler: NextApiHandler): NextApiHandler {
     }
   };
 }
+
+// non-null assertions are ok because users must have an email, and also are guaranteed to be logged in by protect
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
This is cursed, but I wasn't able to find a different solution...someone pls help :)

Reply:
what's the error you're getting? i deleted your comment and then did a `yarn build` and the only warnings i saw were these:
<img width="1051" alt="Screen Shot 2021-10-16 at 3 16 33 PM" src="https://user-images.githubusercontent.com/22948432/137599598-854dcfd4-fc12-4d34-919a-51d65148b318.png">